src/main/resources/digital/slovensko/autogram/server/server.yml

```diff
@@ -583,6 +586,12 @@ components:
           default: sm
           description: "Optional width of the signing document visualization. Values are sm (640px), md (768px), lg (1024px), xl (1280px), xxl (1536px). The minimum visualization width is set to 640px. If the preferred visualization width is exceeds width of client's screen, the visualization width is set to the width of the client's screen."
 
+        fsFormId:
+          type: string
+          default: null
+          example: DPFOBv23/1.0
+          description: Specific identifier for financnasprava.sk EForms. For example, DPFOBv23/1.0 is an identifier of Danove priznanie. The version suffix is optional since no form with version differnet than 1.0 exists at the moment.
```

Comment:
```suggestion
          description: Specific identifier for financnasprava.sk EForms. For example, DPFOBv23/1.0 is an identifier of Danove priznanie. The version suffix is optional since no form with version different than 1.0 exists at the moment.
```